src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -1,80 +1,75 @@
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
 import SidebarNavSign from '../SidebarNavSign';
 // @ts-ignore Module is written in Flow
 import FeatureProvider from '../../common/feature-checking/FeatureProvider';
+// @ts-ignore Module is written in Flow
+import messages from '../messages';
+// @ts-ignore Module is written in Flow
+import localize from '../../../../test/support/i18n.js';
+
+jest.unmock('react-intl');
 
 describe('elements/content-sidebar/SidebarNavSign', () => {
     const onClickRequestSignature = jest.fn();
     const onClickSignMyself = jest.fn();
+    const signLabel = localize(messages.boxSignRequestSignature.id);
+    const requestSignatureButtonText = localize(messages.boxSignRequestSignature.id);
+    const signMyselfButtonText = localize(messages.boxSignSignMyself.id);
 
     const renderComponent = (props = {}, features = {}) =>
         render(
             <FeatureProvider features={features}>
                 <SidebarNavSign {...props} />
             </FeatureProvider>,
+            {
+                wrapper: ({ children }: { children?: React.ReactNode }) => (
+                    <IntlProvider locale="en-US">{children}</IntlProvider>
+                ),
+            },
         );
 
-    test.each([true, false])('should render sign button', isRemoveInterstitialEnabled => {
-        const features = {
-            boxSign: {
-                isSignRemoveInterstitialEnabled: isRemoveInterstitialEnabled,
-            },
-        };
+    test('should render sign button', () => {
+        renderComponent();
 
-        const wrapper = renderComponent({}, features);
-        expect(wrapper.getByTestId('sign-button')).toBeVisible();
+        expect(screen.getByLabelText(signLabel)).toBeVisible();
```

Comment:
is this supposed to be a labelText or a Text? should be able to do `screen.getByText("whatever the string is supposed to be")`, it is recommended if you could do `getByRole` though

Reply:
indeed, updated with `getByRole`